pkg/collect/host_run.go

```diff
@@ -118,6 +136,9 @@ func (c *CollectHostRun) Collect(progressChan chan<- interface{}) (map[string][]
 		if werr, ok := err.(*exec.ExitError); ok {
 			runInfo.ExitCode = strings.TrimPrefix(werr.Error(), "exit status ")
 			runInfo.Error = stderr.String()
+		} else if err == context.DeadlineExceeded {
+			runInfo.ExitCode = "0"
```

Comment:
Should't we use `werr.ExitCode()`? I'd prefer not "swallowing" the error code.

Reply:
There is no Exit Code as as the error is about context becomes done prior to the command is run. Do you think we should update the artificial Exit Code to something else? E.g. `124`?

Reply:
many thanks @banjoh this is done.